### DISCUSSION TITLE
How do I Change the Class or CSS of a Slider Based on the Value?

### DISCUSSION THREAD

**Discussion by VehementHam** [elkowar/eww#1054] [Q&A]
I want to change the min-width of the metric highlight to be 0px only when the value is 0, and otherwise be 6px.

**Comment by VehementHam** [comment]
I figured it out. I created new widgets for the volume and brightness. I created a variable for each of their classes. Then I created two classes called `zero` and `non-zero`. In each of the shell scripts for the volume and brightness, I created a condition that updaeted the value of the class variables based off of the value of the brightness or volume.

**Reply by Soilad** [reply]
can u like paste it here?
